intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -496,6 +496,71 @@ \subsection{Getting module details}
 If you're not sure what all of this means: don't worry, you don't have to know;
 just load the module and try to use the software.
 
+\section{Getting system information about the cluster}
+
+\subsection{Checking the general status of the cluster}
+
+To check the general system state, check \url{https://www.vscentrum.be/en/user-portal/system-status}.
+This has information about scheduled downtime, status of the system, \ldots
+
+To check how much jobs are running in what queues, you can use the \lstinline|qstat -q|
+command:
+
+\begin{prompt}
+%\shellcmd{qstat -q}%
+Queue            Memory CPU Time Walltime Node  Run Que Lm  State
+---------------- ------ -------- -------- ----  --- --- --  -----
+default            --      --       --      --    0   0 --   E R
+q72h               --      --    72:00:00   --    0   0 --   E R
+long               --      --    72:00:00   --  316  77 --   E R
+short              --      --    11:59:59   --   21   4 --   E R
+q1h                --      --    01:00:00   --    0   1 --   E R
+q24h               --      --    24:00:00   --    0   0 --   E R
+                                               ----- -----
+                                                337  82
+\end{prompt}
+
+Here, there are 361 jobs running on the ``long'' queue, and 77 jobs queued. We
+can also see that the ``long'' queue allows a maximum wall time of 72 hours.
+
+\ifgent
+\subsection{Getting cluster state}
+
+You can check \url{http://hpc.ugent.be/clusterstate} to see information about
+the clusters: you can see the nodes that are down, free, partially filled with jobs,
+completely filled with jobs, \ldots.
+
+You can also get this information with a command: \lstinline|pbsmon|:
```

Comment:
`You can also get this information in text form (per cluster separately) with the \lstinline|pbsmon| command:`